spec/support/fixtures/bookshelf/config/environment.rb

```diff
@@ -28,6 +28,7 @@ module Admin
       manifest         "assets.json"
       prefix           "/assets/admin"
       compile          true
+      nested           true
```

Comment:
```suggestion
      nested           false
```

Reply:
I addressed both of these in a single commit (ffbc598)